test/client/nodes_test.rb

```diff
@@ -41,7 +41,7 @@
   it "gets the hot threads for the node(s)" do
     str = $client.nodes.hot_threads :read_timeout => 2
     assert_instance_of String, str
-    assert_match %r/cpu usage by thread/, str
+    assert_match %r/:::/, str
```

Comment:
`Client#hot_threads` returns just the body, so I checked that and not the HTTP status.